src/ApiGenerator/Program.cs

```diff
@@ -40,18 +32,29 @@ private static async Task Main(string[] args)
 			if (string.IsNullOrEmpty(downloadBranch))
 				downloadBranch = DownloadBranch;
 
+			var generateCode = Ask("Generate code from the specification files on disk?", true);
+			var lowLevelOnly = generateCode && Ask("Generate low level client only?", false);
+
 			if (redownloadCoreSpecification)
 				RestSpecDownloader.Download(downloadBranch);
 
-			answer = "invalid";
+			if (generateCode)
+				await Generator.ApiGenerator.Generate(downloadBranch, lowLevelOnly, "Core", "XPack");
+		}
+
+		private static bool Ask(string question, bool defaultAnswer = true)
+		{
+			var answer = "invalid";
+			var defaultResponse = defaultAnswer ? "y" : "n";
+
 			while (answer != "y" && answer != "n" && answer != "")
 			{
-				Console.Write("Generate code from the specification files on disk? [Y/n] (default Y): ");
+				Console.Write($"{question}[y/N] (default {defaultResponse}): ");
```

Comment:
nitpick: The default answer in `[y/n]` should be the capitalized one

Reply:
Good call out will take this on board when introducing command line arguments 👍